node/src/Pos/Launcher/Runner.hs

```diff
@@ -218,8 +219,9 @@ runServer mkTransport mkReceiveDelay mkL (OutSpecs wouts) withNode afterNode oq
     stdGen <- liftIO newStdGen
     logInfo $ sformat ("Our verInfo: "%build) ourVerInfo
     node mkTransport mkReceiveDelay mkConnectDelay stdGen bipPacking ourVerInfo defaultNodeEnvironment $ \__node ->
-        NodeAction mkListeners' $ \converse ->
-            let sendActions :: SendActions m
+        NodeAction mkListeners' $ \__converse ->
+            let converse    = withFailureCleanup oq __converse
```

Comment:
Oh so it does...  hmm. Ok, then I don't understand yet. we should not need to reset in two places.